### ISSUE TITLE
Getting a misleading warning while running CI

### ISSUE THREAD

**Issue by maneetgoyal** [SmartsquareGmbH/delete-old-packages#11]
Warning:

![Screenshot 2021-04-09 at 3 25 03 PM](https://user-images.githubusercontent.com/25290212/114163407-c7dfe900-9947-11eb-9ad2-8ac7032cde34.png)

CI code:

```
      - name: Delete old packages # Ref: https://github.com/marketplace/actions/delete-old-packages
        uses: SmartsquareGmbH/delete-old-packages@v0.3.0
        with:
          organization: xxxx
          keep: 4
          names: |
            xxxx
            xxxx
            xxxx
            xxxx
```

**Comment by rubengees** [comment]
The new `organization` param is missing from the `action.yaml`, will fix!

**Comment by rubengees** [comment]
Should be fixed in [0.3.1](https://github.com/SmartsquareGmbH/delete-old-packages/releases/tag/v0.3.1).
